lib/logstash/outputs/email.rb

```diff
@@ -247,7 +247,11 @@ def receive(event)
         mail.add_file(fileLocation)
       end # end @attachments.each
       @logger.debug? and @logger.debug("Sending mail with these values : ", :from => mail.from, :to => mail.to, :cc => mail.cc, :subject => mail.subject)
-      mail.deliver!
+      begin
+        mail.deliver!
+      rescue Exception => e
```

Comment:
maybe this should be StandardError? and also print out the event itself if debug is enabled maybe?